tests/test_quantum_gradients.py

```diff
@@ -279,48 +279,36 @@ def circuit(x, y):
         assert grad_A == pytest.approx(grad_F, abs=tol)
         assert grad_A2 == pytest.approx(grad_F, abs=tol)
 
-    @pytest.mark.parametrize("name", qml.ops._cv__ops__)
-    def test_CVOperation_with_heisenberg_and_no_params(self, name, gaussian_dev, tol):
-        """An integration test for CV gates that support analytic differentiation
```

Comment:
I realised that the if statement in the test reduces all CV ops loaded to `InterfermoterUnitary` and I make the test more explicit.

Reply:
This is part of a pattern we enforce in a previous PR: do not automatically import and instantiate gates!

Reply:
Useful to know, thanks!